connectors/opa/lib/opa_reader.go

```diff
@@ -22,130 +22,195 @@ func NewOpaReader(opasrvurl string, client *http.Client) *OpaReader {
 	return &OpaReader{opaServerURL: opasrvurl, opaClient: client}
 }
 
-func (r *OpaReader) updatePolicyManagerRequestWithResourceInfo(in *taxonomymodels.PolicyManagerRequest, catalogMetadata map[string]interface{}) (*taxonomymodels.PolicyManagerRequest, error) {
-	responseBytes, errJSON := json.MarshalIndent(catalogMetadata, "", "\t")
+func (r *OpaReader) updatePolicyManagerRequestWithResourceInfo2(in *taxonomymodels.PolicyManagerRequest, catalogMetadata *datacatalogTaxonomyModels.DataCatalogResponse) (*taxonomymodels.PolicyManagerRequest, error) {
+	// just printing - start
+	responseBytes1, errJSON := json.MarshalIndent(catalogMetadata.ResourceMetadata, "", "\t")
```

Comment:
Why do you have 1 and 2 in names of variables and functions?

Reply:
fixed. Thanks.